great_expectations/data_context/store/checkpoint_store.py

```diff
@@ -40,13 +40,16 @@ class CheckpointStore(ConfigurationStore):
     _configuration_class = CheckpointConfig
 
     @override
-    def ge_cloud_response_json_to_object_dict(self, response_json: Dict) -> Dict:
+    def ge_cloud_response_json_to_object_dict(
+        self, response_json: Dict
+    ) -> Union[Dict, List[Dict]]:
```

Comment:
I don't love this union - any way to get around this? If we want to support retrieving and parsing entire collections, we need to support a collection return type.

Reply:
Can we define a separate method for operations against whole collections?